Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift

```diff
@@ -122,7 +122,7 @@ class MSAnalyticsViewController: UITableViewController, AppCenterProtocol {
       for targetToken in MSTransmissionTargets.shared.transmissionTargets.keys {
```

Comment:
MSTransmissionTargets?

Reply:
This is the class of our test app, not SDK